Luke_Smith_Portfolio_About_Me.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
Consider moving your separate html pages into their own folders, e.g. Luke_Smith_Portfolio_About_Me/Luke_Smith_Portfolio_About_Me.html
Having multiple .html files in the same folder can confuse things.